json/overview.json

```diff
@@ -8,7 +8,7 @@
             "release_date": "2018-09-04T09:46:34.743219",
```

Comment:
Now each and every panel has a field named `release_date`. This date needs to be newer than the previous one to get the panel updated. This prevents regressions and allow us to know which version is deployed. Kidash automatically sets this field to current timestamp, but if you are manually modifying the panel, just write current date using the same format, for instance the next line will work:
```
"release_date": "2018-09-05T15:00:00.743219"
```

Thanks!